src/models/PrimaryAuth.js

```diff
@@ -64,6 +68,7 @@ function (Okta, BaseLoginModel, CookieUtil, Enums) {
           remember = this.get('remember'),
           lastUsername = this.get('lastUsername'),
           multiOptionalFactorEnroll = this.get('multiOptionalFactorEnroll');
+      var deviceFingerprintEnabled = this.settings.get('features.deviceFingerprinting');
```

Comment:
nit: let's follow the pattern for the previous variables and not have a new var statement